packages/components/src/components/combobox/shadow.tsx

```diff
@@ -245,7 +245,7 @@ export class KolCombobox implements ComboboxAPI {
 												data-index={index}
 												tabIndex={-1}
 												role="option"
-												aria-selected={this.state._value === option}
+												aria-selected={this.state._value === option ? 'true' : 'false'}
```

Comment:
```suggestion
												aria-selected={this.state._value === option ? 'true' : undefined}
```

Ich bin mir nicht sicher, ob es in der Praxis mit `"false"` ein Problem gibt, aber der Default-Value hier sollte `undefined` sein.

So sind auch die WCAG-Beispiele umgesetzt: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/
Siehe auch: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected#values